component/publisher-client/src/main/java/org/wso2/am/analytics/publisher/client/AbstractMoesifClient.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
```

Comment:
shall we update this with the most recent license header?
Check other places as well